pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -846,6 +846,10 @@ Block.Question:
           minimum: 1
           nullable: true
           description: the number to label this block, `null` if it shouldn't be numbered
+        columnSpan:
+          type: integer
+          minimum: 1
+          description: The number of columns this question should occupy in a tabular display
```

Comment:
To me, it sounds like a violation of the clean code principle. Why should Block.Question know anything about tabular structure? This is an easy way of column-span implementation, but Block.Question is used not only in tabular blocks, right?

Reply:
We have been using the blocks as structural display elements. In some cases they also serve as wrappers for other elements. Current Question block definition below.
I don't see any engineering principles being violated here, but if you have an idea that would be cleaner without overcomplicating things, do suggest. Requirement here is that we need to know how many columns in the question has to display its content.
![Screen Shot 2022-05-27 at 8 29 52 AM](https://user-images.githubusercontent.com/29984380/170699795-b57e36cc-ee45-4237-bfb0-3a8528fb895b.png)